### PR TITLE
Fixes railings mentioning tables on inspect.

### PR DESCRIPTION
**Pull request by 1080pCat**
## What Does This PR Do

image

## Why It's Good For The Game

image

## Testing

Checked if it worked
## Changelog
:cl:
fix: Fixed railings mentioning tables in the tip of click-dragging
/:cl:

### CODE REVIEW

**Review by lewcc — APPROVED** [review]

File: code/game/objects/structures.dm [hunk 1]
```diff
@@ -165,7 +165,7 @@
 		if(examine_status)
 			. += examine_status
 	if(climbable)
-		. += "<span class='info'>You can <b>Click-Drag</b> someone to [src] to put them on the table after a short delay.</span>"
+		. += "<span class='info'>You can <b>Click-Drag</b> someone to [src] to put them on [src] after a short delay.</span>"
```
Comment (CinnamonSnowball) [inline code/game/objects/structures.dm:168]: This should probably use a "it" pronoun, repeating the same thing twice is a bit redundant

**Review by Contrabang — CHANGES_REQUESTED** [review]
You cant actually put people ON a railing, I think you need to put the examine behind a proc on /obj/structure, which returns the text. the railing override could just say you can click drag to climb over it

File: code/game/objects/structures/railings.dm [hunk 2]
```diff
@@ -12,6 +12,9 @@
 	var/currently_climbed = FALSE
 	var/mover_dir = null
 
+/obj/structure/railing/get_climb_text()
+	return "<span class='info'>You can <b>Click-Drag</b> someone to [src] to put them over [src.p_them()] after a short delay.</span>"
```
Comment (Contrabang) [inline code/game/objects/structures/railings.dm:16]: You can't actually click drag people onto railings or tables, only yourself. I think we should probably change these two texts

**Review by S34NW — CHANGES_REQUESTED** [review]
Please address reviews

**Comment by 1080pCat** [comment]
> Please address reviews

Entirely forgot to actually make the change, sorry

**Review by Sirryan2002 — APPROVED** [review]

File: code/game/objects/structures.dm [hunk 3]
```diff
@@ -154,6 +154,9 @@
 		return FALSE
 	return TRUE
 
+/obj/structure/proc/get_climb_text()
+	return "<span class='info'>You can <b>Click-Drag</b> someone to [src] to put them on [src.p_them()] after a short delay.</span>"
```
Comment (Contrabang) [inline code/game/objects/structures.dm:158]:
```suggestion
	return "<span class='info'>You can <b>Click-Drag</b> yourself to [src] to climb on top of [src.p_them()] after a short delay.</span>"
```

**Review by Contrabang — CHANGES_REQUESTED** [review]

File: code/game/objects/structures.dm [hunk 4]
```diff
@@ -154,6 +154,9 @@
 		return FALSE
 	return TRUE
 
+/obj/structure/proc/get_climb_text()
+	return "<span class='info'>You can <b>Click-Drag</b> yourself to [src] to climb on top of [src.p_them()] after a short delay.</span>"
```
Comment (lewcc) [inline code/game/objects/structures.dm:158]: We don't really need pronoun code here
```suggestion
	return "<span class='info'>You can <b>Click-Drag</b> yourself to [src] to climb on top of it after a short delay.</span>"
```

File: code/game/objects/structures/railings.dm [hunk 5]
```diff
@@ -12,6 +12,9 @@
 	var/currently_climbed = FALSE
 	var/mover_dir = null
 
+/obj/structure/railing/get_climb_text()
+	return "<span class='info'>You can <b>Click-Drag</b> yourself to [src] to climb over [src.p_them()] after a short delay.</span>"
```
Comment (lewcc) [inline code/game/objects/structures/railings.dm:16]:
```suggestion
	return "<span class='info'>You can <b>Click-Drag</b> yourself to [src] to climb over it after a short delay.</span>"
```

**Review by lewcc — APPROVED** [review]

**Review by S34NW — APPROVED** [review]

**Pull request merged**